app/templates/buyers/studios_start_page.html

```diff
@@ -48,6 +48,10 @@ <h2>Create a shortlist</h2>
 
         <h2>Contact your shortlist</h2>
         <ol class="govuk-list govuk-list--number" start="5">
+            <li>
```

Comment:
Contact the Crown Commercial Service to get contact details for the suppliers on your shortlist.

Reply:
Thanks Nora, I've updated with your suggested content